packages/core/src/utils/typenames.ts

```diff
@@ -42,9 +41,7 @@ const hasTypenameField = (set: SelectionSetNode) => {
 };
 
 const formatNode = (node: FieldNode | InlineFragmentNode) => {
-  if (!node.selectionSet) {
-    return false;
```

Comment:
The return false here isn’t required I believe hit it does stop `visit` from traversing the rest of the AST. Not sure if that even makes a difference here though

Reply:
There isn't much to traverse further if it has no selectionSet so I think removing it is acceptable